src/governance/classes.cpp

```diff
@@ -492,10 +494,20 @@ CAmount CSuperblock::GetPaymentsLimit(int nBlockHeight)
         return 0;
     }
 
+    bool fMNRewardReallocated = llmq::utils::IsMNRewardReallocationActive(::ChainActive().Tip());
```

Comment:
nit: there's three calls of ::ChainActive().Tip() - maybe better to save it in tmp variable with `pindex` ?

also do we need lock for `cs_main` here? need annotations for `GetPaymentsLimit` about mutex.

Reply:
`::ChainActive()` locks `cs_main` internally